css/style.css

```diff
@@ -0,0 +1,1544 @@
+}
+
+/* styling for large screen */
+
+/* top navigation styling */
+.navtop {
+  background-color: black;
+  padding: 10px;
+  position: relative;
+}
+
+.navlist {
+  display: flex;
+  list-style-type: none;
+}
+
+.navlist li {
+  margin-right: 15px;
+}
+
+.navlist a {
+  text-decoration: none;
+  color: #aaa;
+}
+
+.footbottom a {
+  text-decoration: none;
+  color: rgba(167, 169, 169, 1);
+}
+
+.footnav a {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: rgba(167, 169, 169, 1);
+  margin-right: 10px;
+}
+
+.flexed-desc a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-third a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-second a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-first a {
+  text-decoration: none;
+}
+
+.textfoot a {
+  font-family: "Varela Round", sans-serif;
+  font-size: 12px;
+  color: rgba(36, 36, 36, 0.9);
+}
+
+.title a {
+  font-family: "Russo One", sans-serif;
+  text-decoration: none;
+  color: #f42;
+  font-size: 24px;
+  font-weight: bolder;
+}
+
+.titleandlinks {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 20px;
+  align-items: baseline;
+}
+
+.titleholder {
+  margin-right: auto;
+}
+
+.title-side {
+  margin: 0 6px;
+}
+
+.p-holder {
+  display: flex;
+}
+
+.a-side {
+  text-decoration: none;
+  border-right: 1px solid gray;
+  padding-right: 3px;
+  font-family: "Russo One", sans-serif;
+  color: #f42;
+}
+
+.item a {
+  text-decoration: none;
+  color: black;
+  font-size: 16px;
+  font-family: "Varela Round", sans-serif;
+  font-weight: bold;
+}
+
+.yellow-nav a {
+  margin-left: 15px;
+  text-decoration: none;
+  color: black;
+  font-size: 15px;
+}
+
+.dealtitlediv p a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.navlist a:hover {
+  color: white;
+}
+
+#whitetext {
+  color: white;
+}
+
+/* navlogom&s style */
+.navlogom-m {
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 106px;
+  height: 55px;
+  text-align: center;
+  padding-top: 25px;
+  display: none;
+  top: 0;
+  left: 0;
+}
+
+.navlogom-m img {
+  width: 80%;
+}
+
+.navlogom-s {
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 85px;
+  height: 25px;
+  padding-top: 15px;
+  padding-left: 20px;
+  display: none;
+}
+
+/* navigation fixed styling */
+.header {
+  position: sticky;
+  top: 0;
+  z-index: 100;
+}
+
+.navfixed {
+  height: 50px;
+  font-family: "Nova Square", cursive;
+  font-size: 17px;
+  background-color: white;
+}
+
+#sideinfologo {
+  width: 125px;
+  height: 54px;
+}
+
+.sideinfo {
+  margin-right: 25px;
+}
+
+.time {
+  margin-right: 20px;
+  font-size: 120%;
+  color: red;
+  font-weight: bold;
+}
+
+.ticket {
+  background-color: #0cc;
+  color: white;
+  width: 90px;
+  height: 30px;
+  font-size: 60%;
+  border: none;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.ticket:hover {
+  background-color: purple;
+}
+
+#searchbtn {
+  transform: rotate(90deg);
+  color: #aaa;
+}
+
+#hamburger {
+  display: none;
```

Comment:
- [ ] OPTIONAL ➡ `display: none` is repeated 20+ times throughout your code. I recommend you guys to create a `.d-none {display: none}` class to handle that rule, in order to keep your DRY code.